CONTRIBUTING.md

```diff
@@ -272,6 +272,8 @@ Akka generates class diagrams for the API documentation using ScalaDoc.
 Links to methods in ScalaDoc comments should be formatted
 `[[Like#this]]`, because `[[this]]` does not work with genjavadoc, and
 IntelliJ warns about `[[#this]]`.
+For further hints on how to disambiguate links in scaladoc comments see
+[this StackOverflow answer](https://stackoverflow.com/a/31569861/354132).
```

Comment:
"but note that all Scaladoc link syntax will not be possible since the docs also has to be valid Javadoc" or something along those lines?

Reply:
perhaps we should eventually improve genjavadoc to perform some conversion there... :thinking:

Reply:
I think it won't be possible to do 100% of scaladoc anyways because type system? I don't have a concrete example of the top of my head though.